src/modules/commandrunner/tests/CommandRunnerTests.cpp

```diff
@@ -127,6 +127,7 @@ namespace Tests
 
         EXPECT_EQ(MMI_OK, m_commandRunner->Get(m_component, m_reportedObject, &reportedPayload, &payloadSizeBytes));
         EXPECT_TRUE(IsJsonEq(Command::Status::Serialize(status), std::string(reportedPayload, payloadSizeBytes)));
+        delete[] reportedPayload;
```

Comment:
>delete[] reportedPayload; [](http://example.com/codeflow?start=8&length=25)

reportedPayload is a std::string which when going out of scope, its destructor automatically frees memory. Can you please double-check on this? Same for the other similar instances in this file.

Reply:
Good catch! Please go ahead with the proposed change.